test/integration/targets/filesystem/tasks/setup.yml

```diff
@@ -17,7 +17,7 @@
       - ansible_os_family != 'Suse'
       - not (ansible_distribution == 'Ubuntu' and ansible_distribution_version is version('16.04', '<='))
       - ansible_system != "FreeBSD"
-      - not (ansible_distribution == "RedHat" and ansible_distribution_major_version|int >= 8)
+      - not (ansible_facts.os_family == "RedHat" and ansible_facts.distribution_major_version is version('8', '>='))
```

Comment:
Is there a reason to use `ansible_facts.os_family` instead of `ansible_os_family` ?

Reply:
Planning for the future when `inject_facts_as_vars` is not enabled by default. Mostly personal preference, though, as I like to access the data from within the `ansible_facts` dictionary rather than the copy. My mind is weird. :)